en/css/README.md

```diff
@@ -297,7 +297,7 @@ Look at the code we just pasted to find the places where we added classes in the
 
 Don't be afraid to tinker with this CSS a little bit and try to change some things. Playing with the CSS can help you understand what the different things are doing. If you break something, don't worry – you can always undo it!
 
-We really recommend taking this free online [Codeacademy HTML & CSS course](https://www.codecademy.com/tracks/web). It can help you learn all about making your websites prettier with CSS.
+We really recommend taking this free online [freeCodeCamp - Basic HTML & HTML5](https://learn.freecodecamp.org/). It can help you learn all about making your websites prettier with CSS.
```

Comment:
What is free and online? I'm not a native English speaker but would assume, that you loose the `course` word.

Reply:
Yeah, it looks like that word got lost.

Reply:
sorry, do i need to add the word and resubmit?

Reply:
@fernandasj to merge your changes here they need to be accepted. So you need to resolve whatever issue is here, commit that and push it to the same branch. Your updates will occur here automatically.

Reply:
Great! Can you also answer a question posted below by @das-g?